Frontend/ui-driver/src/Flow.purs

```diff
@@ -1832,7 +1853,9 @@ homeScreenFlow = do
       myRidesScreenFlow
     GO_TO_REFERRAL_SCREEN_FROM_HOME_SCREEN -> do
       liftFlowBT $ logEvent logField_ "ny_driver_rankings"
-      referralScreenFlow
+      let cityConfig = getCityConfig appConfig.cityConfig (getValueToLocalStore DRIVER_LOCATION)
```

Comment:
update here as well

Reply:
and if this block of code is same and repetitive then create a common function to handle this flow